rq/worker.py

```diff
@@ -122,7 +122,7 @@ def find_by_key(cls, worker_key, connection=None):
 
     def __init__(self, queues, name=None,
                  default_result_ttl=None, connection=None,
-                 exc_handler=None, default_worker_ttl=None, job_class=None):  # noqa
+                 exception_handlers=None, default_worker_ttl=None, job_class=None):  # noqa
```

Comment:
We should provide a deprecation path for `exc_handler` and display a warning message if `exc_handler` is used. Something like this would be good:
```python
if exception_handlers is not None:
    for handler in exception_handlers:
        self.push_exc_handler(handler)

else:
    self.push_exc_handler(self.move_to_failed_queue)
    if exc_handler is not None:
         self.push_exc_handler(exc_handler)
        # Display warning and ask user to use `exception_handlers` instead

```